src/CMakeLists.txt

```diff
@@ -32,8 +33,8 @@ target_compile_definitions(
   # change log levels concurrently across threads.
   SPDLOG_NO_ATOMIC_LEVELS
   # Defines a macro which is used to construct the URL paths to individual
-  # endpoint handlers (e.g. /irods-http/1.0.0/collections).
-  IRODS_HTTP_API_BASE_URL="/irods-http/${IRODS_HTTP_API_VERSION}")
+  # endpoint handlers (e.g. /irods-http-api/0.0.1/collections).
```

Comment:
0.0.1

Reply:
My original plan with this PR was to reset the version to 0.0.1.

It shows this is its own thing and provides plenty of room to change things on the way to 1.0.0.

At the moment, this has been superseded by the iRODS version. So we need to figure out which version scheme is more appropriate. Should this follow the plugin versioning scheme or its own like metalnx?

Reply:
leaning into 'client' argument... so not tied to irods version.

Reply:
Removed ties to iRODS server version. It is now set to 0.1.0.

Done.